ydb/core/blobstorage/vdisk/common/vdisk_hugeblobctx.h

```diff
@@ -71,11 +71,13 @@ namespace NKikimr {
         // check whether this NEW blob is huge one; userPartSize doesn't include any metadata stored along with blob
         bool IsHugeBlob(TBlobStorageGroupType gtype, const TLogoBlobID& fullId) const;
 
-        THugeBlobCtx(ui32 minREALHugeBlobInBytes, const std::shared_ptr<const THugeSlotsMap> &hugeSlotsMap, bool addHeader)
-            : MinREALHugeBlobInBytes(minREALHugeBlobInBytes)
+        THugeBlobCtx(ui32 minHugeBlobInBytes, ui32 appendBlockSize, const std::shared_ptr<const THugeSlotsMap> &hugeSlotsMap, bool addHeader)
+            : MinREALHugeBlobInBytes(minHugeBlobInBytes / appendBlockSize * appendBlockSize + 1)
```

Comment:
+ 1?

Reply:
Yes, this is the way it works now. So, I am just preserving the current behaviour here.  
Please, see the TAllChains::GetMinREALHugeBlobInBytes